chain/db/table.go

```diff
@@ -156,3 +156,16 @@ type ReportDetail struct {
 	Data         []byte `gorm:"not null"`
 	Exitcode     uint8  `gorm:"not null"`
 }
+
+type Packet struct {
+	Type        string          `gorm:"not null"`
```

Comment:
Let's not name this field `type`. It will complicate frontend.

Reply:
I implemented example code for frontend to get this field already